docs/gettinginvolved/functional-testing.rst

```diff
@@ -134,7 +134,49 @@ To run the functional tests, run the following command:
 OpenStack
 =========
 
-The configuration stanza for an private OpenStack deployment is the same as Rackspace above, but ``auth_plugin`` should be included, which refers to an authentication plugin provided by ``python-keystoneclient``.
+The configuration stanza for a private OpenStack deployment is similar to Rackspace (above), with a few notable differences:
+
+* ``auth_plugin`` should be included, which refers to an authentication plugin provided by ``python-keystoneclient``, and
+* ``provider: "openstack"`` should be included, if the top level key is not ``openstack``.
 
 If required, you may need to add additional fields.
 For more information, see :ref:`openstack-dataset-backend`.
+
+Devstack
+--------
+
+It is assumed that you have a working ``Devstack`` environment.
+Refer to document "Setting up a devstack instance" on Google Drive.
```

Comment:
We do this elsewhere:

```
$ git grep "Google Drive"
docs/gettinginvolved/functional-testing.rst:Refer to document "Setting up a DevStack instance" on Google Drive.
docs/gettinginvolved/infrastructure/release-process.rst:* Access to ClusterHQ's Google Drive for access to ClusterHQ versioning policy documents.
docs/gettinginvolved/infrastructure/release-process.rst:   - Refer to the appropriate internal release planning document on Google Drive for a list of features that were scheduled for this release, e.g. Product > Releases > Release 0.3.1, and add bullet points for those features that have been completed.

```

I guess we move those DevStack notes into our Flocker docs, but not in this branch.